cfpb/college-costs
# Functional tests

This adds @marteki's functional test work with just a teeny bit of clean up and bug fixin' from me.
## To test

Pull this branch down and run `npm install`

Then make sure you have webdriver and protractir all set up:

``` bash
npm install -g webdriver-manager protractor
webdriver-manager update --standalone
```

To run the tests:

``` bash
# start webdriver in a terminal tab:
webdriver-manager start
# in a fresh tab:
protractor test/functional/conf.js
```
## Note

This requires a Node version 4+ or greater. We should either make note of this or use older versions of the testing modules to support older versions of Node. This isn't likely to be an issue anywhere other than a Jenkins server.
## Review
- @higs4281 
- @marteki 
- @niqjohnson
